source/includes/inkling-reference/_lessons.html.md

```diff
@@ -137,8 +137,6 @@ lessonStatement ::=
     testClause?
 ```
 
-Select the Syntax tab to see the lesson syntax. The syntax for lesson and its subordinate
```

Comment:
What was the reason for making this change to the lessons file in this PR instead of the future lessons PR? I'd like to keep the changes isolated if possible.